package.json

```diff
@@ -351,6 +351,11 @@
     "webpack-hot-middleware": "^2.18.0",
     "webpack-subresource-integrity": "^1.0.0-rc.1"
   },
+  "storybook-deployer": {
+    "gitUsername": "echo $GH_USER",
+    "gitEmail": "echo $GH_EMAIL",
```

Comment:
I am not too sure about to refer to these vars (assuming these are set in travis?)

Reply:
can we use the defaults?

Reply:
yeh I can remove this if we are leaving the defaults.